### ISSUE TITLE
[Bug] Input array validations not working at all in forms

### ISSUE THREAD

**Issue by ManfrediPist** [Semantic-Org/Semantic-UI#6368]
Coming from [[Form Validation] Add Data-Validate Example #2683](https://github.com/Semantic-Org/Semantic-UI/issues/2683) I'm struggling with the same problem since, as another user pointed out, the problem hasn't been fixed at all :/

### Steps
Setting proper name and date-validate for each input they should be parsed and validated properly according to the doc.

### Expected Result
If any of the input didn't pass the validation the form should print the error message we are used to.
### Actual Result
Only the first input get validated, the other pass the validation in any case 
### Version
2.3.1

### Testcase
http://jsfiddle.net/92p8fkbb/

**Comment by prudho** [comment]
@hammy2899 this is more a bug than an enhancement. The rule function seem to test three times the first value instead of testing each value one time.

I'll submit a PR shortly to fix this.

**Comment by hammy2899** [comment]
@prudho I see where you're coming from but the validation doesn't have that functionality therefor it would need to be added hence the enhancement.

I will take a look at your PR when I get some time today.

**Comment by ManfrediPist** [comment]
@prudho Hey, I've been testing your fix but it seems not to work properly. When there are multiple array input, each time you try to validate they get validated as a block, what I mean is :

Input1 ->good
Input2 ->bad
Input3 ->bad

I click on "validate button" and also the input1 get marked as bad .

**Comment by prudho** [comment]
Yeah, I see what you mean. Actually I don't know how to deal with that, since the three inputs are handled together in the code...

@hammy2899 you were right, this is more an enhancement than a bug. My PR can only validate all fields of a group, but can't show which field is invalid... Fell free to reject it if it doesn't help at all.

**Comment by lubber-de** [comment]
Individual field error handling for input array formfields is now fixed in https://github.com/fomantic/Fomantic-UI/pull/387
See http://jsfiddle.net/a1cnph4f/1/
Supposed to be part of upcoming Fomantic-UI 2.7,2
